syntax = "proto3";

package hello_world;

//import "google/protobuf/any.proto";

// The greeting service definition.
service Greeter {
// Sends a greeting
rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
    string name = 1;
  //  google.protobuf.Any payload = 2;
}

// The response message containing the greetings
message HelloReply {
    string message = 1;
  //  google.protobuf.Any payload = 2;
}
